client/src/plus/offer-overview/offer-overview-subscribe/index.tsx

```diff
@@ -174,7 +174,14 @@ function OfferDetails({
             <a href={ctaLink} className="sub-link">
               {offerDetails.upgradeCta}
             </a>
-          )) || <span className="sub-link na">Not available</span>}
+          )) || (
+            <a
+              href="/en-US/plus/faq#can-i-upgrade/downgrade-my-plan-"
```

Comment:
This anchor link doesn't seem to work? Maybe double check

Reply:
Did you restart `yarn dev` or run `yarn tool spas` manually? Otherwise that Markdown isn't converted.

Reply:
Never mind, you're right. We seem to have a bug in our heading id generator. It doesn't replace slashes. 🤦

Reply:
```
-/en-US/plus/faq#can-i-upgrade/downgrade-my-plan-
+/en-US/plus/faq#can-i-upgrade-downgrade-my-plan-
```

Reply:
Actually, that's not even the problem!

This is only a problem locally, as pages are loaded asynchronously and the hash is not applied again after the page was loaded. On stage/prod the page is loaded directly.

Reply:
@Guyzeroth For example, if you visit http://localhost:3000/en-US/docs/Learn/HTML#solving_common_html_problems locally, you will see that you don't end up at the right position.

Reply:
🎯 Tracking in https://github.com/mdn/yari/issues/5733.

Reply:
Alright great thanks!